core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -35,28 +35,28 @@
 import hudson.model.listeners.ItemListener;
 import hudson.remoting.VirtualChannel;
 import hudson.util.FormValidation;
-import java.io.File;
-
+import jenkins.MasterToSlaveFileCallable;
+import jenkins.model.BuildDiscarder;
+import jenkins.model.Jenkins;
+import jenkins.tasks.SimpleBuildStep;
+import jenkins.util.BuildListenerAdapter;
+import net.sf.json.JSONObject;
 import org.apache.tools.ant.types.FileSet;
-import org.kohsuke.stapler.StaplerRequest;
```

Comment:
Ok, legitimate question - Intellij is going to reorder this crap in some way so long as it's set to handle imports for me. Unsurprisingly, our imports are chaotic as is. Should I really be worrying about this or just shrug and move on, accepting the diffs?